cms/urls.py

```diff
@@ -141,6 +141,8 @@
     # rest api for course import/export
     path('api/courses/', include('cms.djangoapps.contentstore.api.urls', namespace='courses_api')
          ),
+    path('api/import_from_modulestore/',
+         include('cms.djangoapps.import_from_modulestore.views.urls', namespace='import_from_modulestore_api')),
```

Comment:
```suggestion
    path('api/import_from_modulestore/',
         include('cms.djangoapps.import_from_modulestore.rest_api.urls', namespace='import_from_modulestore_api')),
```

To be consistent with other edx-platform apps, the `import_from_modulestore/views/` folder should be called `import_from_modulestore/rest_api/`